ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2024b.yaml

```diff
@@ -0,0 +1,317 @@
+  from: intel-base-gpu-ubi9-python-3.11
+  to: jupyter-intel-tensorflow-ubi9-python-3.11
+- context_dir: intel/runtimes/pytorch/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: intel-base-gpu-ubi9-python-3.11
+  to: intel-runtime-pytorch-ubi9-python-3.11
+- context_dir: jupyter/intel/pytorch/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: intel-base-gpu-ubi9-python-3.11
+  to: jupyter-intel-pytorch-ubi9-python-3.11
+- context_dir: intel/runtimes/ml/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.11
+  to: intel-runtime-ml-ubi9-python-3.11
+- context_dir: jupyter/intel/ml/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.11
+  to: jupyter-intel-ml-ubi9-python-3.11
+- context_dir: base/c9s-python-3.11
+  dockerfile_path: Dockerfile
+  to: base-c9s-python-3.11
+- context_dir: codeserver/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.11
+  to: codeserver-ubi9-python-3.11
+- context_dir: rstudio/c9s-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-c9s-python-3.11
+  to: rstudio-c9s-python-3.11
+- context_dir: cuda/c9s-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-c9s-python-3.11
+  to: cuda-c9s-python-3.11
+- context_dir: rstudio/c9s-python-3.11
+  dockerfile_path: Dockerfile
+  from: cuda-c9s-python-3.11
+  to: cuda-rstudio-c9s-python-3.11
+- context_dir: rocm/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.11
+  to: rocm-ubi9-python-3.11
+- context_dir: jupyter/minimal/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-ubi9-python-3.11
+  to: rocm-jupyter-minimal-ubi9-python-3.11
+- context_dir: jupyter/datascience/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-jupyter-minimal-ubi9-python-3.11
+  to: rocm-jupyter-datascience-ubi9-python-3.11
+- context_dir: jupyter/rocm/tensorflow/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-jupyter-datascience-ubi9-python-3.11
+  to: rocm-jupyter-tensorflow-ubi9-python-3.11
+- context_dir: jupyter/rocm/pytorch/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-jupyter-datascience-ubi9-python-3.11
+  to: rocm-jupyter-pytorch-ubi9-python-3.11
+- context_dir: runtimes/rocm-pytorch/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-ubi9-python-3.11
+  to: runtime-rocm-pytorch-ubi9-python-3.11
+- context_dir: runtimes/rocm-tensorflow/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  from: rocm-ubi9-python-3.11
+  to: runtime-rocm-tensorflow-ubi9-python-3.11
+promotion:
+  to:
+  - namespace: red-hat-data-services
+    tag: 2024b-weekly
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: notebook-jupyter-minimal-ubi9-python-3-11-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: jupyter-minimal-ubi9-python-3.11
+    env:
+      IMAGE_REPO: odh-minimal-notebook-container
+      IMAGE_TAG: YearIndex
+      RELEASE_VERSION: v3-2024b
```

Comment:
How is this value determined? I don't see any logic when comparing values from the other files here?

Reply:
Are you referring to `V3`? If so, the naming convention follows this pattern:

v1 -> Python 3.8
v2 -> Python 3.9
v3 -> Python 3.11

The Vx naming convention dates back to the early days of ODH. It’s just a label and could be anything, as the real differentiation comes from the YYYYx part of the tag.

For example, the current tag is:
`quay.io/modh/odh-minimal-notebook-container:v2-2024a-20240829-76a016f`

It might be clearer to use a more descriptive tag, such as:
`quay.io/modh/odh-minimal-notebook-container:ubi9-py3.11-2024b-20240829-76a016f`

This would make it easier to identify key components like the OS and Python version at a glance.

What do you thing? 

cc @harshad16

Reply:
Okay, I can see it now - we have this value different - for some images we use this format, for others we use some long name instead, see the following files and search for `RELEASE_VERSION:`
* https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-main.yaml
* https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2024a.yaml
* https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2023b.yaml
* https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2023a.yaml
* https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-1.2.yaml

So, it's not a change, thus no objections anymore. Just probably slightly confusing and maybe we should consider to unify this.